doc/source/user/basics.rec.rst

```diff
@@ -185,6 +186,15 @@ The string representation of a structured datatype is shown in the "list of
 tuples" form if possible, otherwise numpy falls back to using the more general
 dictionary form.
 
+The dtype object can also be accessed with both keys and values individual:
+
+>>> d = np.dtype([('x', 'i8'), ('y', 'f4')])
+>>> for key in d.names:
+...     value = d[key]
+...     print(key, value)
+x int64
+y float32
```

Comment:
How about a middle ground: moving this snippet to line 169?

Reply:
IMO the example around L166 already covers this and is more idiomatic. In principle one could add a line demonstrating `d.fields` as well, which gives this info directly.

Reply:
@rossbar buddy, can you tell me what would be the possible way that I can include this code snippet?